NitroxPatcher/Patches/Dynamic/CyclopsSonarButton_Update_Patch.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    public class CyclopsSonarButton_Update_Patch : NitroxPatch, IDynamicPatch
```

Comment:
Do you know which logic is causing the sonar ping to not be visible when you exit the cyclops while another pilot is still driving?

What about adding a comment here? Something like this?
```suggestion
    /// <summary>
    /// Prevents sonar from turning off automatically when the player stops piloting the Cyclops.
    /// The sonar will stay on until the player leaves the vehicle and automatically turns on when they enter again (if sonar was on at that time). 
    /// </summary>
    public class CyclopsSonarButton_Update_Patch : NitroxPatch, IDynamicPatch
```